blockbase/inc/social-navigation.php

```diff
@@ -13,11 +13,6 @@ function blockbase_condition_to_render_social_menu( $block_content, $block ) {
 		return false;
 	}
 
-	// The block should have an unstable location attribute.
```

Comment:
Considering the social navigation render override only happens when an `__unstableLocation` is present none of this code will run.  I think we can completely remove the `social-navigation.php` file.